### PR TITLE
DMS changing versions

### PR DESCRIPTION
**Pull request by temandr**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -737,12 +737,11 @@
         ],
         "dms-preview": [
             {
-                "downloadUrl": "https://azcliorcas.blob.core.windows.net/azclipath/dms_preview-0.8.1-py2.py3-none-any.whl",
-                "filename": "dms_preview-0.8.1-py2.py3-none-any.whl",
```
Comment (tjprescott) [inline src/index.json]: You should keep the old entry so there is a version that can be installed on older versions of the CLI.

Comment (limingu) [reply src/index.json]: Either you can keep the old entry. Or you can keep "azext.minCliCoreVersion": "2.0.43" and remove max cli core version. As the new code with min_profile can work with all core versions

Comment (vchske) [reply src/index.json]: I would go with removing max and keeping the original min.

Comment (temandr) [reply src/index.json]: Updated the version the original version.

**Comment by tjprescott** [comment]
Please resolve the conflicts and get the CI passing.

**Pull request merged**